tests/src/system/packages/MessageHubFeedTests.scala

```diff
@@ -113,4 +134,71 @@ class MessageHubFeedTests
     runActionWithExpectedResult(actionName, "dat/multipleValueTypes.json", expectedOutput, false)
   }
 
+  it should "fire a trigger when a binary message is posted to message hub" in withAssetCleaner(wskprops) {
```

Comment:
Would be nice if we had a generic way to create a trigger, produce a value, and check that value from the fired trigger. Such a mechanism would easily allow us to run this end-to-end type test with minimal duplicate code as it is now in three different files.